ansible/roles/provision-vm/tasks/redhat.yml

```diff
@@ -24,6 +24,19 @@
       delay: 30
       until: registration is not failed
 
+- name: Register server with Red Hat for IBM POWER vms
```

Comment:
This is duplicated with the same block above - we can set the block to run on both s390x and ppc64le to avoid this duplication

Reply:
yes  I have made it common for ppc64le and s390 x by adding condition 
when: arch_info == "ppc64le" or arch_info == "s390x"